### PR TITLE
K8SPSMDB-328 add custom params for pmm-admin

### PR DESCRIPTION
**Pull request by hors**
[K8SPSMDB-328](https://jira.percona.com/browse/K8SPSMDB-328)

### CODE REVIEW

File: pkg/controller/perconaservermongodb/psmdb_controller.go [hunk 1]
```diff
@@ -652,7 +652,11 @@ func (r *ReconcilePerconaServerMongoDB) reconcileMongos(cr *api.PerconaServerMon
 			return errors.Wrapf(err, "check pmm secrets: %s", usersSecretName)
 		}
 
-		pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec)
+		customAdminParams := ""
+		if cr.Spec.PMM.MongosParams != "" {
+			customAdminParams = cr.Spec.PMM.MongosParams
+		}
+		pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec, customAdminParams)
```
Comment (blez) [inline pkg/controller/perconaservermongodb/psmdb_controller.go:659]:
```
pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec, cr.Spec.PMM.MongosParams)
```

File: pkg/controller/perconaservermongodb/psmdb_controller.go [hunk 2]
```diff
@@ -849,8 +853,11 @@ func (r *ReconcilePerconaServerMongoDB) reconcileStatefulSet(arbiter bool, cr *a
 			if err != nil {
 				return nil, fmt.Errorf("check pmm secrets: %v", err)
 			}
-
-			pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec)
+			customAdminParams := ""
+			if cr.Spec.PMM.MongodParams != "" {
+				customAdminParams = cr.Spec.PMM.MongodParams
+			}
+			pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec, customAdminParams)
```
Comment (blez) [inline pkg/controller/perconaservermongodb/psmdb_controller.go:860]:
```
pmmC, err := psmdb.AddPMMContainer(cr, usersSecretName, pmmsec, cr.Spec.PMM.MongodParams)
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: pkg/controller/perconaservermongodb/psmdb_controller.go (hunk 1 above)
Comment (hors) [reply pkg/controller/perconaservermongodb/psmdb_controller.go]: done

File: pkg/controller/perconaservermongodb/psmdb_controller.go (hunk 2 above)
Comment (hors) [reply pkg/controller/perconaservermongodb/psmdb_controller.go]: done

**Review by delgod — APPROVED** [review]

**Review by blez — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| self-healing | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | failed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| demand-backup-sharded | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |

**Pull request merged**